README.md

```diff
@@ -58,6 +58,31 @@ try {
 }
 ````
 
+If the values from `EvaluationResponse` class doesn't matter and only a Color is needed:
+````java
+flagr = new Flagr(FLAGR_HOST);
+Color color = flagr.evaluateAndGetVariantAttachment(
+        new EvaluationContext("color"),
+        Color.class,
+        new Color("white", "#FFFFFF") // default value returned when the flag is disabled or there's some connection problem.
+);
+````
+
+When only the variant matters, use:
+````java
+flagr = new Flagr(FLAGR_HOST);
+String variant = flagr.evaluateVariant(new EvaluationContext("myflag")));
+````
+
+And if it's just a simple on/off flag. Use the `evaluateBoolean` method. For this the
+variants configured on Flagr UI needs to correspond to true/false or enabled/disabled:
```

Comment:
I would explain it like this: 

> returns `true` if the variant evaluates to "true", "enabled" or "on" otherwise it returns `false`.

Reply:
Switched to your version. I also changed the method to match it.